packages/transactions/src/TransactionService.tsx

```diff
@@ -101,13 +114,42 @@ const Transactions: FC = () => {
   };
 
   return (
-    <RemoteDataTable<Transaction>
-      columns={columns}
-      onFetchData={fetchData}
-      onRowClick={row => {
-        navigate(`/customers/customer-invoice/${row.id}`);
-      }}
-    />
+    <>
+      <Portal container={appBarButtonsRef.current}>
+        <>
+          <Button
+            startIcon={<Book />}
+            onClick={() =>
+              (window.location.href = 'https://docs.msupply.foundation')
+            }
+          >
+            {t('button.docs')}
+          </Button>
+          <Button
```

Comment:
as this point I wondered about creating a simpler to use `Button` component: just pass in the LocaleKey and the icon and the onClick handler 🤷‍♂️

Reply:
Ah yup.. good idea

Maybe we could think about integrating muis `Typography` components and  FormatJS 🤷 

The extraction in formatjs seems nice also